lms/djangoapps/course_api/views.py

```diff
@@ -123,6 +124,24 @@ def get_object(self):
         )
 
 
+class CourseListUserThrottle(UserRateThrottle):
+    """Limit the number of requests users can make to the course list API."""
```

Comment:
One small nit on the comment, can we add the JIRA ticket as context to why we added this Throttle?

Reply:
@staubina Done.